metadata/org.ehcache/ehcache/3.10.8-jakarta/reflect-config.json

```diff
@@ -0,0 +1,5168 @@
+    },
+    "name": "com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.ehcache.xml.multi.XmlMultiConfiguration"
+    },
+    "name": "com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.ehcache.xml.ResourceConfigurationParser"
+    },
+    "name": "com.sun.org.apache.xerces.internal.impl.dv.xs.ExtendedSchemaDVFactoryImpl",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.ehcache.xml.ConfigurationParser"
+    },
+    "name": "com.sun.org.apache.xerces.internal.impl.dv.xs.SchemaDVFactoryImpl",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.ehcache.xml.ResourceConfigurationParser"
+    },
+    "name": "com.sun.org.apache.xerces.internal.impl.dv.xs.SchemaDVFactoryImpl",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.ehcache.xml.XmlUtil"
+    },
+    "name": "com.sun.org.apache.xerces.internal.impl.dv.xs.SchemaDVFactoryImpl",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.ehcache.xml.multi.XmlMultiConfiguration"
+    },
+    "name": "com.sun.org.apache.xerces.internal.impl.dv.xs.SchemaDVFactoryImpl",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.ehcache.xml.ConfigurationParser"
+    },
+    "name": "com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderFactoryImpl",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.ehcache.sizeof.impl.AgentLoader"
+    },
+    "name": "com.sun.tools.attach.VirtualMachine",
+    "methods": [
+      {
+        "name": "attach",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "detach",
+        "parameterTypes": []
+      },
+      {
+        "name": "loadAgent",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.ehcache.xml.XmlConfiguration"
+    },
+    "name": "jakarta.xml.bind.annotation.adapters.CollapsedStringAdapter",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
```

Comment:
- @wilkinsona  I added for `jakarta.xml.bind:jakarta.xml.bind-api:3.0.1` and `org.glassfish.jaxb:jaxb-runtime:3.0.2` GraalVM reachability metadata by changing `user-code-filter.json` and removed all GraalVM reachability metadata related to MBeans.
- But once this entry is removed, `nativeTest` fails, should I also move it under `test/resources/META-INF/native-image`?

Reply:
So, I don't know. I'm not very familiar with the infrastructure that's used to test the metadata.

Reply:
- I noticed https://github.com/oracle/graalvm-reachability-metadata/blob/0.2.5/metadata/org.hibernate.orm/hibernate-core/6.1.1.Final/reflect-config.json#L1725 , this entry also exists, which seems to be a common occurrence. 

- Of course, this does not affect, I manually removed all entries related to `jakarta.xml.bind.**` in the latest commit. This entry now belongs in the `org.glassfish.jaxb:jaxb-runtime:3.0.2` separate GraalVM reachability metadata.
```json
[
 {
     "condition": {
       "typeReachable": "org.glassfish.jaxb.core.v2.ClassFactory"
     },
     "name": "jakarta.xml.bind.annotation.adapters.CollapsedStringAdapter",
     "methods": [
       {
         "name": "<init>",
         "parameterTypes": []
       }
     ]
   }
]
```